.github/workflows/cd-reset-internal-testnet.yml

```diff
@@ -40,6 +40,7 @@ jobs:
       - name: take the chain offline
         run: bash ${GITHUB_WORKSPACE}/.github/scripts/put-all-chain-nodes-on-standby.sh
         env:
+          CHAIN_TAG_NAME: ${{ inputs.chain-tag-name }}
```

Comment:
thanks for making this more configurable for the different approaches. can we give it a clearer name? maybe `AWS_CHAIN_ID_TAG_NAME`